ProfileView/package.json

```diff
@@ -26,22 +26,22 @@
     "lit-html": "1.2.1"
   },
   "peerDependencies": {
-    "@cjaas/common-components": "2.2.1",
-    "@momentum-ui/core": "19.15.4",
+    "@cjaas/common-components": "3.0.0",
+    "@momentum-ui/core": "19.15.10",
     "@momentum-ui/icons": "8.21.0",
     "@momentum-ui/utils": "6.2.15",
-    "@momentum-ui/web-components": "2.4.2",
+    "@momentum-ui/web-components": "2.4.7",
```

Comment:
I'm getting a peer dependency issue when trying to start with this . . . let's just make both `"@momentum-ui/web-components"` 2.4.8

Reply:
I have it updated in the latest PR.